apps/site/navigation.json

```diff
@@ -20,6 +20,11 @@
       "link": "https://nodejs.org/docs/latest/api/",
       "label": "components.containers.navBar.links.docs"
     },
+    "contribute": {
```

Comment:
OOC -- what is the point of a contribute page within the website if we simply redirect here? Should we then simply delete the page and make the link on the sidebar also redirect here? I'm fine with that TBH and possibly is the best thing to be done here, can you do that?

Reply:
I can certainly do that, but as we've learned recently, this is a [CONTENT VS CODE decision](https://github.com/nodejs/nodejs.org/blob/main/CONTENT_VS_CODE.md).

I've audited the content on https://nodejs.org/en/about/get-involved/contribute, comparing it to https://github.com/nodejs/nodejs.org/blob/main/CONTENT_VS_CODE.md. I want to make sure we are not net-losing content that may not be present within https://github.com/nodejs/node/blob/main/CONTRIBUTING.md


| website section | comparative location on main repo | plan |
| -- | -- | -- |
| [asking for general help](https://nodejs.org/en/about/get-involved/contribute#asking-for-general-help) | [linked on the doc](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#issues) | ✅ already handled |
| [reportng an issue, general](https://nodejs.org/en/about/get-involved/contribute#reporting-an-issue) | [linked on the doc](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#issues) | ✅ already handled |
| reporting an issue specific to the website | nothing | do nothing, expect to triage and move issues? |
 | reporting and issue, [website status](https://status.nodejs.org/) link | nothing | non-blocking in my opinion,  [already on the README of nodejs.org](https://github.com/nodejs/nodejs.org). Add to [org-level README](https://github.com/nodejs/.github/blob/main/profile/README.md) or [nodejs/node README](https://github.com/nodejs/node) as other work  |
| [Code contributions](https://nodejs.org/en/about/get-involved/contribute#code-contributions) | [linked on the doc](https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md) | ✅ already handled|
| [node todo link](https://www.nodetodo.org/) | [differing content](https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#setting-up-your-local-environment) | ✅ already handled |
| [becoming a collaborator](https://nodejs.org/en/about/get-involved/contribute#becoming-a-collaborator) | [linked to, found within GOVERNANCE.md](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#contributing-to-nodejs) | ✅ already handled|

Reply:
Technically speaking we wouldn't be authoring content; But removing it, which IMO we've done plenty of times before 👀

Reply:
resolved via https://github.com/nodejs/nodejs.org/pull/7274/commits/4093ba29370def1c77c8e92e2a763df154a7d76c